code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -899,28 +899,70 @@
 
 /datum/reagent/medicine/earthsblood //Created by ambrosia gaia plants
 	name = "Earthsblood"
-	description = "Ichor from an extremely powerful plant. Great for restoring wounds, but it's a little heavy on the brain."
+	description = "Ichor from an extremely powerful plant. Some conspiracy theorists INSISTS this plant was used by Centcom during Plasmastock to minimize any chance of rioting."
 	color = rgb(255, 175, 0)
 	overdose_threshold = 25
+	taste_description = "shag carpeting and rock"
+	var/ODcycle = 0 //adds to current_cycle in calculation. 5 cycles while OD = 1 "normal" cycle (basically you're getting 1.2 instead of 2 cycles during OD)
 
 /datum/reagent/medicine/earthsblood/on_mob_life(mob/living/carbon/M)
-	M.adjustBruteLoss(-3 * REM, 0)
-	M.adjustFireLoss(-3 * REM, 0)
-	M.adjustOxyLoss(-15 * REM, 0)
-	M.adjustToxLoss(-3 * REM, 0)
-	M.adjustOrganLoss(ORGAN_SLOT_BRAIN, 2 * REM, 150) //This does, after all, come from ambrosia, and the most powerful ambrosia in existence, at that!
-	M.adjustCloneLoss(-1 * REM, 0)
-	M.adjustStaminaLoss(-30 * REM, 0)
-	M.jitteriness = min(max(0, M.jitteriness + 3), 30)
+	if(current_cycle <= 20) //5u has to be processed before u get into THE FUN ZONE
+		M.adjustBruteLoss(-1 * REM, 0)
+		M.adjustFireLoss(-1 * REM, 0)
+		M.adjustOxyLoss(-0.5 * REM, 0)
+		M.adjustToxLoss(-0.5 * REM, 0)
+		M.adjustCloneLoss(-0.1 * REM, 0)
+		M.adjustStaminaLoss(-0.5 * REM, 0)
+	else
+		M.adjustBruteLoss(-5 * REM, 0)
+		M.adjustFireLoss(-5 * REM, 0)
+		M.adjustOxyLoss(-3 * REM, 0)
+		M.adjustToxLoss(-3 * REM, 0)
+		M.adjustCloneLoss(-1 * REM, 0)
+		M.adjustStaminaLoss(-3 * REM, 0)
+		M.jitteriness = min(max(0, M.jitteriness + 3), 30)
+		metabolization_rate += 0.2
 	M.druggy = min(max(0, M.druggy + 10), 15) //See above
 	..()
 	. = 1
 
 /datum/reagent/medicine/earthsblood/overdose_process(mob/living/M)
 	M.hallucination = min(max(0, M.hallucination + 5), 60)
 	M.adjustToxLoss(5 * REM, 0)
+	ODcycle += 0.2
 	..()
 	. = 1
+/datum/reagent/medicine/earthsblood/on_mob_end_metabolize(mob/living/L)
+	var/mob/living/carbon/human/hippie = L
+	if(!istype(L))
+		return ..()
+	var/resi = NONE
+	switch(current_cycle + ODcycle)
+		if(0-20) //i'm a (benevolent) god sweet nerevar
```

Comment:
0 to 20, also having tested this it always kicks out as a permanent trauma. Tested the iteration previous to this and the trauma was always serious (resilience 2), though in that case it did not look to be the fault of the switch case.